test/framework/src/main/java/org/elasticsearch/test/rest/support/Features.java

```diff
@@ -34,7 +34,7 @@
  */
 public final class Features {
 
-    private static final List<String> SUPPORTED = Arrays.asList("stash_in_path", "groovy_scripting", "headers", "embedded_stash_key");
+    private static final List<String> SUPPORTED = Arrays.asList("stash_in_path", "groovy_scripting", "headers", "embedded_stash_key", "yaml");
```

Comment:
well well if you were an external contributor I would run the whole suite that takes 45 minutes. But in your case, trust won. I should have totally counted the characters of that line as part of the review.

P.S. we java folks are the first ones pushing without running tests at times (WAT?), so no biggie. I will check closer next time.